pkg/server/sandbox_run.go

```diff
@@ -180,10 +180,24 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 			id, err)
 	}
 
-	// Add sandbox into sandbox store.
-	meta.CreatedAt = time.Now().UnixNano()
+	// Setup network.
 	// TODO(random-liu): [P2] Replace with permanent network namespace.
 	meta.NetNS = getNetworkNamespace(createResp.Pid)
+	if err = c.netPlugin.SetUpPod(meta.NetNS, config.GetMetadata().GetNamespace(), name, id); err != nil {
```

Comment:
Add unit test for this. The unit test of `RunPodSandbox` is simple now, just assert `SetUpPod` is called with correct namespace, name and id.

We'll need to test cleanup in future PRs.